### PR TITLE
fix(readme): update korean readme

### PR DESCRIPTION
**Pull request by rabbits99**
I uploaded again
sorry for make you confused

### CODE REVIEW

File: .github/i18n/README.kr.md [hunk 1]
```diff
@@ -406,22 +405,21 @@ Search Pokemon with state-of-the-art visual representation
...
-
+Jina는 오픈소스 프로젝트이다. 우리는 풀스택 개발자, evangelists, 프로젝트 매니저들을 [채용](https://jobs.jina.ai)하여 뉴럴 탐색 생태계를 오픈소스에 구축하려고 한다.
 
-## 면허증
+## 라이선스
 
-저작권 (c) 2020 지나 AI 유한회사. 모든 권리 보유.
+Copyright (c) 2020 Jina AI Limited. All rights reserved
 
-Jina는 Apache License 버전 2.0에 따라 라이센스를 받는다.[전체 라이센스 텍스트는 LICE를 참조하십시오.](LICENSE)
```
Comment (JoanFM) [inline .github/i18n/README.kr.md:427]: I would suggest to keep this part in original English, at least the `Apache License` part.

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by JoanFM** [comment]
Bad commit message, please refer to:
- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
#553 :bookmark: Commit Message Guideline for the First Time Contributor
jina-ai/jina | Jun 21st | Added by GitHub
CONTRIBUTING.md
jina-ai/jina | Added by GitHub

**Comment by JoanFM** [comment]
recheckcla

**Comment by bwanglzu** [comment]
hi @rabbits99 would you like to continue with this PR? it has been 2 weeks. Please check commit guidelines #553 to update your commit message, and sign the CLA.

**Comment by rabbits99** [comment]
yes sorry for keeping wait.

this is sign for cla
I have read the CLA Document and I hereby sign the CLA

**Comment by bwanglzu** [comment]
@rabbits99 no worries, you need to amend this commit message, otherwise ci will keep failing: 
image

**Comment by rabbits99** [comment]
we have trouble with amending 
so can we create PR again?

**Comment by nan-wang** [comment]
> we have trouble with amending
> so can we create PR again?

Yes, it is easier to create a PR again.

**Pull request closed without merging**